src/Notification/Notification.module.scss

```diff
@@ -0,0 +1,107 @@
+@import '../mixins.scss';
+@import '../variables.scss';
+
+.wrapper {
+    @include text-normal;
+
+    border-radius: $border-radius;
+    padding: 14px 18px 16px 15px;
+    width: 320px;
+    margin-bottom: 15px;
+}
+
+.default {
+    color: var(--grey-dark-2);
+    background-color: var(--white);
+    fill: var(--black);
+
+    .title {
+        color: var(--black);
+    }
+
+    .closeIcon {
+        fill: var(--grey-dark);
+    }
```

Comment:
As discussed in the previous PR, this kind of `css selected` is bad and is completely unnecessary in this situation.
`.title` and `.closeIcon` should be nested